lib/systemd/system/delphix-migration.service

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[Unit]
+Description=Delphix OS Migration Service
+After=systemd-udev-settle.service
```

Comment:
can you leave a comment describing why we're using these `After=` and `Before=` directives? I presume it's important, so it'd be good to describe their importance here; otherwise, why not use something more standard like `After=local-fs.target`?